Makefile.core.mk

```diff
@@ -293,8 +293,13 @@ lint: lint-python lint-copyright-banner lint-scripts lint-go lint-dockerfiles li
 
 .PHONY: check-agent-deps
 check-agent-deps:
-	@go list -f '{{ join .Deps "\n" }}' ./pkg/bootstrap |\
-		(! grep -P 'k8s.io/api/|k8s.io/apiextensions-apiserver|k8s.io/client-go|sigs.k8s.io/gateway-api')
+	@go list -f '{{ join .Deps "\n" }}' \
+			./security/pkg/nodeagent/caclient/... \
```

Comment:
Maybe we should pull in goda then we can do `goda tree 'agent=1(reach(./pilot/cmd/pilot-agent:all, k8s.io/client-go/pkg/apis/clientauthentication/v1))'` (and similar) instead of needing to try to find all the package agent imports?.

Or if there is a more native way

https://github.com/loov/goda

Reply:
Oh, I know where the problems are. This will become pilot/cmd/pilot-agent eventually, but I need finer grained to preserve the incremental progress.

Reply:
Solving some of the problems will require some re-thinking - e.g. Delta SDS and wasm using RBAC. We'll tackle them gradually.